### DISCUSSION TITLE
Newbie in trying to execute the Android Remote Control Client on my S8 tablet

### DISCUSSION THREAD

**Discussion by Phazz34** [deckerth/FoobarRemoteController2#3] [Q&A]
First i apologyze for requesting some help but i must admit i have explored everything i could to make the app working, but i failed.

As per required, i successfully installed

- Beefweb Remote Control (foo_beefweb) successfully installed and configured
- check with my IP_adress : 8880 done and fully operational

When executing the Android Remote Control Client and configuring my IP_adress >> nothing happens at all

Foobar2000 is executed and running on my PC desktop

Any help please or advise on what i could miss ??

Thx a lot

**Comment by deckerth** [comment]
Can you access the beefweb Interface with the Browser?

Phazz34 ***@***.***> schrieb am Do., 19. Dez. 2024, 15:01:

**Reply by Phazz34** [reply]
Yes indeed i access the web interface

**Comment by deckerth** [comment]
Sorry, but then I am a bit clueless at the Moment. stay tuned.. The new
completely reworked Version will be out soon.

Phazz34 ***@***.***> schrieb am Sa., 21. Dez. 2024, 15:10:

**Comment by Coleogo** [comment]
In my "Xiaomi 12 lite" it works only if WiFi is on and SIM data is off. As DATA is turned on, the connection with the computer is lost.
I had put the IP of the desktop with automatic ports abilitation into the firewall.
Maybe this is the right way to use FoobarLink, and the only that works for me.

**Comment by deckerth** [comment]
I think that accessing foobar with mobile data is an external access. I do
not think that your foobar server is accessible from outside. Open a
browser and try to open http://<ip-address>:8880/api this will probably not
work with mobile data.

Am Do., 16. Jan. 2025 um 21:38 Uhr schrieb Coleogo ***@***.***

**Reply by Coleogo** [reply]
I only meant to help Phazz34 bringing my personal experience.
I never tried to connect to Foobar through mobile data.
I noticed this "issue" searching for lyrics in the meantime I was listening to music using the remote.
I wanted to reply directly to Phazz34 but there is no "Write a reply" box under Phazz34's post, so it's my fault.
Thank you anyway for your fast response.

**Comment by Phazz34** [comment]
Thanks a lot for your investigations and proposition.
Nevertheless none of them are helpful. My Galaxy S8 is pure wifi there is no SIM for data connexion

**Comment by deckerth** [comment]
We will ship an update in the next days.

**Comment by deckerth** [comment]
Does this work meanwhile?
